package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-commercetools",
-  "version": "3.2.2",
+  "version": "3.3.0",
```

Comment:
@PhilippSpo this requires a change in code. Most people(e.g boilerplate) use `^` and a minor releases means that it we don't need to update the package.json to get this new version. Getting this version without updating the code will cause errors. Get my point?